gems/activerecord/6.0/activerecord-generated.rbs

```diff
@@ -16651,7 +16651,7 @@ module ActiveRecord
     #
     #   Post.find_by_sql ["SELECT title FROM posts WHERE author = ? AND created > ?", author_id, start_date]
     #   Post.find_by_sql ["SELECT body FROM comments WHERE author = :user_id OR approved_by = :user_id", { :user_id => user_id }]
-    def find_by_sql: (untyped sql, ?untyped binds, ?preparable: untyped? preparable) { () -> untyped } -> untyped
+    def find_by_sql: (untyped sql, ?untyped binds, ?preparable: untyped? preparable) ?{ () -> untyped } -> untyped
```

Comment:
Looks good. Could you move this definition to activerecord.rbs, please?
We use that file to manage manually modified definitions.

Reply:
My bad, forgot about the naming convention between the files :sweat_smile:

Reply:
Let me know if I should also remove the large code comment above the method signature. I was unsure about that.

Reply:
Good change! It would be better to move both signature and comment :-)